### PR TITLE
other: Ignore formatting commits in git blame

### PR DESCRIPTION
**Pull request by algochoi**
Adds a `.git-blame-ignore-revs` to exclude formatting commits in the git history. This makes it easier to trace the git blame from source code. To enable locally, you may have to edit your Git settings: `git config blame.ignoreRevsFile .git-blame-ignore-revs`.

Visually confirmed in the Github blame viewer on this branch - there are some occasional lines that slip through, but I think it's a problem with the UI.

### CODE REVIEW

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# .git-blame-ignore-revs
+# Exclude formatting commits made by black
+a9facdf30ca467b6e16d13ce88d8889bba7f2e1c
+ea81ba259216b8be7fb86ef5d3e9c151a94edf87
+81fe44f0dd9785b1665bbbea047aed591912534b
```
Comment (tzaffi) [inline .git-blame-ignore-revs:5]:
```suggestion
a9facdf30ca467b6e16d13ce88d8889bba7f2e1c   # more black formatting Aug 17, 2021
ea81ba259216b8be7fb86ef5d3e9c151a94edf87   # formatting Aug 16, 2021 
81fe44f0dd9785b1665bbbea047aed591912534b  # apply black Aug 11, 2021
```

Suggesting that we add a timestamp to aid in quickly assessing how much blame ignore coverage we have

**Review by tzaffi — APPROVED** [review]
Super addition!

Made an ignoreable suggestion

**Comment by algochoi** [comment]
Needed to update the ubuntu image to make sure CI builds pass - related to https://github.com/algorand/indexer/pull/1545

**Review by algobarb — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ jobs:
       python-version:
         type: string
     machine:
-     image: "ubuntu-2004:202104-01"
+     image: "ubuntu-2204:2022.04.2"
```
Comment (excalq) [inline .circleci/config.yml:36]: [Warty Warthog](https://en.wikipedia.org/wiki/Ubuntu_version_history#Ubuntu_4.10_(Warty_Warthog)) calling!

**Review by excalq — APPROVED** [review]

**Pull request merged**